File: backend/micro_epoweredmove_reviews_reservations/src/main/java/com/kosdiam/epoweredmove/models/enums/ReviewStatus.java
package com.kosdiam.epoweredmove.models.enums;

public enum ReviewStatus {
    ACTIVE("Active"),
    CANCELLED("Cancelled");

    private String reviewStatus;
    private ReviewStatus(String currentType){
        this.reviewStatus = reviewStatus;
    }

    @Override
    public String toString(){
        return reviewStatus;
    }
}

File: frontend/E-PoweredMoveAndroidApp/app/src/main/java/com/kosdiam/epoweredmove/models/enums/ReviewStatus.kt
package com.kosdiam.epoweredmove.models.enums

import com.kosdiam.epoweredmove.R
import com.kosdiam.epoweredmove.models.interfaces.IEnumName

enum class ReviewStatus (override val stringResource: Int) : IEnumName {
    ACTIVE(R.string.review_status_active),
    CANCELLED(R.string.review_status_cancelled);
}